packages/agents/cartographer/poller/src/adapters/database/index.ts

```diff
@@ -63,3 +63,13 @@ export const getDatabase = async (): Promise<Database> => {
     getCheckPoint,
   };
 };
+
+export const closeDatabase = async (): Promise<void> => {
+  const { logger } = getContext();
+  try {
```

Comment:
I think no need to try-catch-throw here, the error will just bubble through anyways.

Reply:
Done